bigquery/tests/unit/test_client.py

```diff
@@ -65,19 +75,19 @@ def test__get_query_results_miss_w_explicit_project_and_timeout(self):
 
         creds = _make_credentials()
         client = self._make_one(self.PROJECT, creds)
-        conn = client._connection = _Connection()
+        conn = client._connection = _make_connection()
 
         with self.assertRaises(NotFound):
             client._get_query_results(
                 'nothere', None, project='other-project', timeout_ms=500)
 
-        self.assertEqual(len(conn._requested), 1)
-        req = conn._requested[0]
-        self.assertEqual(req['method'], 'GET')
+        self.assertEqual(len(conn.api_request.call_args_list), 1)
+        req = conn.api_request.call_args_list[0]
+        self.assertEqual(req[1]['method'], 'GET')
         self.assertEqual(
-            req['path'], '/projects/other-project/queries/nothere')
+            req[1]['path'], '/projects/other-project/queries/nothere')
         self.assertEqual(
-            req['query_params'], {'maxResults': 0, 'timeoutMs': 500})
+            req[1]['query_params'], {'maxResults': 0, 'timeoutMs': 500})
```

Comment:
This would likely be clearer / more idiomatic using `assert_called_once_with`, e.g.:

```python
conn.api_request.assert_called_once_with(
    method='GET',
    path='/projects/other-project/queries/nothere',
    query_params={'maxResults': 0, 'timeoutMs': 500},
)
```

Reply:
Done in https://github.com/GoogleCloudPlatform/google-cloud-python/pull/5066/commits/cd1c6b179b5d1b54f20804cf527e06fed8c75eaa.